src/Modal.js

```diff
@@ -156,6 +160,19 @@ const Modal = React.createClass({
     if (this.props.keyboard && e.keyCode === 27) {
       this.props.onRequestHide();
     }
+  },
+
+  focusModalContent () {
+    this.lastFocus = document.activeElement;
```

Comment:
Thank you. 
It is important for me to know about it.
Also I'll be able to do more comprehensive code reviews with regard to `iFrames`.